### PR TITLE
Implement GitHub Actions

### PR DESCRIPTION
**Pull request by proudmuslim-dev**
`gradlew` shows as modified due to me marking it as an executable.
Edit: This PR was redone and merged [here](https://github.com/Skytils/SkytilsMod/pull/26).

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: build-ci
```
Comment (proudmuslim-dev) [inline .github/workflows/gradle.yml:1]: Could probably have a better name for this but I cba to come up with one

**Comment by proudmuslim-dev** [comment]
Let me fix it, forgot to add `setupDecompWorkspace`

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Java setup
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Build
+      run: ./gradlew setupDecompWorkspace && ./gradlew build
```
Comment (My-Name-Is-Jeff) [inline .github/workflows/gradle.yml]: Why not setupCiWorkspace?

Comment (My-Name-Is-Jeff) [inline .github/workflows/gradle.yml]: It doesn't seem to upload artifacts either

**Review by My-Name-Is-Jeff — CHANGES_REQUESTED** [review]

File: .github/workflows/gradle.yml (hunk 2 above)
Comment (proudmuslim-dev) [reply .github/workflows/gradle.yml]: You're right, I'd forgotten

Comment (proudmuslim-dev) [reply .github/workflows/gradle.yml]: Resolved

Comment (proudmuslim-dev) [reply .github/workflows/gradle.yml]: I can fix that as well, one moment

File: .github/workflows/gradle.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Java setup
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
```
Comment (My-Name-Is-Jeff) [inline .github/workflows/gradle.yml]: Is it possible to add the cache-packages task

**Review by My-Name-Is-Jeff — CHANGES_REQUESTED** [review]

**Comment by proudmuslim-dev** [comment]
This will be reopened once it isn't a dumpster fire and I can actually focus

**Pull request closed without merging**